packages/engineer/test/dev-server.unit.ts

```diff
@@ -586,6 +600,22 @@ describe('engineer:dev-server', function () {
         expect(pid1).to.eq(pid2);
         expect(pid1).to.eq(process.pid);
     });
+
+    it('runs app with the correct runtime metadata', async () => {
```

Comment:
please also make a test in scripts, to check that the proper output path is added when running an already built application

Reply:
Added test